README.md

```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
+Example:
+```js
+const config = {
+  apiUrl: 'YOUR_URL',
+  containerId: 'getid-component',
+  flowName: 'yourFlowName',
+  locale: 'en'
+};
+```
 
 
 ### Profile
-Add list availeble fieds for crosschecking
-Add keys with custom fields
-
-
-### Widget object
-
-Example: 
-
-``` js
- const config = {
-   apiUrl: 'YOUR_URL',
-   isPopUp,
- }; 
+We have opportunity to pre-fill the Profile form data by set property *profile* into config:
+example 
+```js
+const config = {
+  apiUrl: 'YOUR_URL',
+  containerId: 'getid-component',
+  flowName: 'yourFlowName',
+  profile: [
+    { category: 'First name', value: 'Jon' },
+    { category: 'Last name', value: 'Dow' },
+    { category: 'Date of birth', value: '2010-10-10' },
+    { category: 'Address', value: 'France le croisan 4-27' },
+    { category: 'Date of expiry', value: '1111-11-11' },
+    { category: 'Date of issue', value: '2021-02-15' },
+    { category: 'Document number', value: '4114414141' },
+    { category: 'Email', value: 'email@email.com' },
+    { category: 'Gender', value: 'male' },
+    { category: 'Nationality', value: 'AZE' },
+    { category: 'Personal number', value: '123123123123' },
+  ],
+};
 ```
+In case when you not specify the Form field but specify this data into profile, this data will send from web sdk to the server and will participate in cross-checking
+
+We provide cross-checking comparison by next Form fields:
+* First name
+* Last name
+* Date of birth
+* Issue country
+* Gender
+* Document type
+* Date of issue
+* Date of expiry
+* Nationality
+* Nationality code
+* Document number
+* Place of birth
+* Place of issue
 
 ### Visual Appearance
 
-SHADOW DOM
+It is possible to customize the styles by two way:
+- By insert the custom css as string
+- By CSS Variables
 
-1. CSS Variables
-2. cssString
 
+To insert the custom css just specify filed *injectCSS* into config with a css string as a value
+example
+```js
+const config = {
+  apiUrl: 'YOUR_URL',
+  containerId: 'getid-component',
+  flowName: 'yourFlowName',
+  injectCSS: 'p {color: red; text-align: center;}'
+}
+```
+The using custom variables it's the more flexible and convenient way
+list of variables:
+```
+--getid-txt-color
+--getid-txt-secondary-color
+--getid-input-bg-color
+--getid-input-active-text-color
+--getid-accent-color
+--getid-second-color
+--getid-input-border-color
+--getid-error-border-color
+--getid-success-border
+--getid-block-border
+--getid-input-hover-border-color
+--getid-input-active-border-color
+--getid-light-color
+--getid-error-color
+--getid-header-size
+--getid-subheader-size
+--getid-font-size
+--getid-font-size-small
+--getid-font-weight
+--getid-font-weight-bold
+--getid-input-width
+--getid-transition
+--getid-font-family
+--getid-background-color
+--getid-max-width
+--getid-min-width
+--getid-border-radius
+--getid-input-border-radius
+```
+to customize style specify this variables for element with next id: getid-main, getid-popup__main
```

Comment:
Use these element ids (`getid-main`, `getid-popup__main`) to customize style using variables